codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
```

Comment:
There is already a similar code-gen config in `graphql.config.json`. Please merge these additions with that file (by modifying the content under `apollo`)